source/connect/advanced-connection-options/advanced-connection.txt

```diff
@@ -25,6 +25,10 @@ Procedure
 .. procedure::
    :style: normal
 
+   .. step:: Open the :guilabel:`New Connection` window.
+
+      .. include:: /includes/open-new-connection.rst
+
    .. step:: Click :guilabel:`Advanced Connection Options`.
 
       .. figure:: /images/compass/advanced-connection-options.png
```

Comment:
[question] Given the new text on step 1, do we need to update this screenshot, too? I didn't see it listed on [DOCSP-37460](https://jira.mongodb.org/browse/DOCSP-37460)

Reply:
Added a note in DOCSP-37460 to add screenshots for the new connection window and the connections sidebar!